### DISCUSSION TITLE
Is there an existing component/library for a wiki/docs/help page with built-in functions like search and indexing?

### DISCUSSION THREAD

**Discussion by egonsung** [reflex-dev/reflex#3924] [Q&A]
Hey guys,

I'm currently building a help page or docs page for my app. I am wondering if I can use any existing libraries rather than building it from scratch. Something similar to the reflex docs page would be great.

Thank you.

**Comment by elviskahoro** [comment]
There actually is something like this that we've built but it's not open sourced yet. I'll try to remember to post back here once it is!

**Reply by egonsung** [reply]
Great to hear that, that's awesome. Looking forward to it.

**Reply by egonsung** [reply]
> There actually is something like this that we've built but it's not open sourced yet. I'll try to remember to post back here once it is!

Hey @elviskahoro, is there an update on this? Thank you!
